pages/project/read/views/sections/downloads.jsx

```diff
@@ -138,7 +144,8 @@ export default function Downloads() {
 
           {
             // latest version might be application, amendment or granted
-            latestVersion && <DownloadSection project={project} version={latestVersion} />
+            latestVersion &&
+              <DownloadSection project={project} version={latestVersion} viewingAtAAEstablishment={viewingAtAAEstablishment} />
```

Comment:
I think for the sake of clarity I would move the check for the `viewingAtAAEstablishment` check to outside the component rather than passing in what is effectively a `dontRenderThisComponent` prop.

Otherwise it is unclear why `viewingAtAAEstablishment` is passed to some instances of `<DownloadSection />` but not others (the answer being that sometimes it should prevent rendering, but not others). Since we're already doing checks outside the component for whether it should render it is clearer to make this part of those checks.